plugins/idea/pluginImpl/src/main/java/com/imaginea/kodebeagle/impl/util/ScalaImportsUtil.java

```diff
@@ -29,9 +30,12 @@
 import com.intellij.psi.util.PsiTreeUtil;
 import java.util.Map;
 import java.util.Set;
-import org.jetbrains.plugins.scala.lang.psi.impl.ScalaFileImpl;
+// import org.jetbrains.plugins.scala.lang.psi.impl.ScalaFileImpl;
+
+public class ScalaImportsUtil {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty class, write its code or make it an "interface". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2094)